geograypher/meshes/meshes.py

```diff
@@ -880,6 +901,7 @@ def save_mesh(self, savepath: PATH_TYPE, save_vert_texture: bool = True):
         Path(savepath).parent.mkdir(parents=True, exist_ok=True)
         # Actually save the mesh
         self.pyvista_mesh.save(savepath, texture=vert_texture)
+        self.save_IDs_to_labels(savepath.parent)
```

Comment:
I'd recommend this be the path to the mesh, with the mesh extension replaced with `_IDs_to_labels.json`

Reply:
I agree, we discussed that for render_labels_pytorch3d, the file can remain as IDs_to_labels.json